rchain_grpc/rho_types.py

```diff
@@ -65,18 +71,24 @@ def _(message: Message) -> dict:
     return {f[0].name: to_dict(f[1]) for f in message.ListFields()}
 
 
-def from_dict(d: dict, klass: GrpcClass) -> GrpcClass:
-    proto = klass()
-    for key, value in d.items():
-        setattr(proto, key, value)
-    return proto
+def from_dict(d: dict, grpc_class: GrpcClass) -> GrpcClass:
```

Comment:
when I wrote it I didn't know about `Klass(x=999)` so I think we can remove this function if we find time for it

Reply:
this can be skipped for now

Reply:
I'll add a note